foundations/tests/tracing.rs

```diff
@@ -18,10 +20,13 @@ fn test_rate_limiter(mut ctx: TestTelemetryContext) {
     assert_eq!(ctx.traces(Default::default()).len(), 10);
 
     ctx.set_tracing_settings(TracingSettings {
-        rate_limit: RateLimitingSettings {
-            enabled: true,
-            max_events_per_second: 5,
-        },
+        sampling_strategy: SamplingStrategy::Active(ActiveSamplingSettings {
```

Comment:
It would be nice to have some basic test for passive sampling

Reply:
Fixed the current test, as it wasn't behaving as expected, and added a simple one for passive sampling